test/Parse/ConditionalCompilation/aarch64AndroidTarget.swift

```diff
@@ -0,0 +1,14 @@
+// RUN: %swift -typecheck %s -verify -D FOO -D BAR -target aarch64-none-linux-android -disable-objc-interop -D FOO -parse-stdlib
```

Comment:
Why the `-D FOO -D BAR`?  Why the duplication of `-D FOO`?

Reply:
I don’t know if there's a reason. It is there in every other file in the same directory. I simply copy/pasted the template.

Reply:
I think that we should drop them for the test.  It is adding noise and just makes it harder to debug things when you try to re-run the commands.

Reply:
@jopamer: long shot, but you added the tests back in 2014 (b46d20240d6e59a9d372f8d1d8fd24cb4af94c4b) with those `-D FOO -D BAR … -D FOO`. Any chance you remember the reason? Would you mind if I remove them from all the target tests in this directory?